scripts/exregional_run_prdgen.sh

```diff
@@ -0,0 +1,437 @@
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 1 ak ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 2 ak ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 3 ak ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 4 ak ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 1 hi ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 2 hi ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 1 pr ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 2 pr ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 1 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 2 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 3 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 4 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 5 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 6 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 7 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 8 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 9 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "$SCRIPTSdir/exregional_run_prdgen_subpiece.sh $fhr $cyc 10 namerica ${DATAprdgen} ${COMOUT} &" >> $DATAprdgen/poescript_${fhr}
+echo "wait" >> $DATAprdgen/poescript_${fhr}
+chmod 775 $DATAprdgen/poescript_${fhr}
+
+#-----------------------------------------
+# Execute the script
+#-----------------------------------------
+
+PREP_STEP
+export CMDFILE=$DATAprdgen/poescript_${fhr}
+eval ${RUN_CMD_PRDGEN} ${CMDFILE} ${REDIRECT_OUT_ERR} || print_err_msg_exit "\
+Call to parallel prdgen for forecast hour $fhr returned with non-
+zero exit code."
+POST_STEP
+
+#----------------------------------------
+# reassemble the output grids
+#----------------------------------------
+
+cat $DATAprdgen/prdgen_conus_1/conus_1.grib2 \
```

Comment:
A similar loop could be applied in this section, too.

Reply:
Agreed, will look into doing that.